apps/suite-launch/test-presets/golden-path-v01-11-2023/fluent-bit.conf

```diff
@@ -0,0 +1,156 @@
+[SERVICE]
+    # See:
+    # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_firelens.html
+    # https://github.com/aws-samples/amazon-ecs-firelens-under-the-hood/tree/master/generated-configs/fluent-bit
+    HTTP_Server  On
+    HTTP_Listen  0.0.0.0
+    HTTP_PORT    2020
+    Flush        1
+    Grace        30
+    Log_Level    info
+    storage.path /var/log/flb-storage/
+
+{{#if segfault}}
+# Artificially trigger a segfault if configured
+[INPUT]
+    Name      exec
+    command   kill -11 `pgrep fluent-bit`
+{{/if}}
+
+{{#if add_forward}}
+
+# This is an artificial forward because my standard out logger was going to some other destination not the cw specified here.
+[INPUT]
+    Name        forward
+    Listen      0.0.0.0
+    Port        6270
```

Comment:
Do we know why? FireLens should handle stdout for us... we should use the FireLens setup so that we are testing it...

Reply:
Hmm, no i don't I tested it for a couple of hours and gave up.

Reply:
Added an action item